static/usage/v8/select/helper-error/index.md

```diff
@@ -0,0 +1,24 @@
+import Playground from '@site/src/components/global/Playground';
+
+import javascript from './javascript.md';
+import react from './react.md';
+import vue from './vue.md';
+
+import angular_example_component_html from './angular/example_component_html.md';
+import angular_example_component_ts from './angular/example_component_ts.md';
+
+<Playground
+  version="8"
+  code={{
```

Comment:
```suggestion
  size="300px"
  code={{
```

Suggestion to make the size a bit larger to show the entire alert in the demo without scrolling